libmscore/chord.cpp

```diff
@@ -2285,11 +2285,9 @@ void Chord::layoutTablature()
                   }
             }
       if (!tab->genDurations()            // if tab is not set for duration symbols
-            || track2voice(track())       // or not in first voice
-            || isGrace()) {               // or chord is a grace (no dur. symbols for graces
```

Comment:
After having some time to think about it, I think, the best would be to enable the possibility to have flags, but per default have them hidden. 
I think that 80% doesn't use grace notes at all, and probably more of the rest doesn't use any flag.
But it is necessary to have the possibility to have flags. Couldn't it be done by an AND beam on Auto?